File: OpenCV/01_fundamentos/python/main.py
import cv2
import numpy as np

img = np.zeros((360, 640, 3), dtype=np.uint8)
img[:, :320] = (255, 0, 0)      # BGR: azul
img[:, 320:] = (0, 180, 255)    # naranja
roi = img[80:280, 220:420]
cv2.rectangle(img, (220, 80), (420, 280), (255, 255, 255), 2)
print("shape=", img.shape, "dtype=", img.dtype, "ROI=", roi.shape)
cv2.imwrite("output_fundamentos.png", img)

File: OpenCV/01_fundamentos/cpp/main.cpp
#include <opencv2/opencv.hpp>
#include <iostream>
int main(){ cv::Mat img(360,640,CV_8UC3,cv::Scalar(0,0,0)); img(cv::Rect(0,0,320,360)).setTo(cv::Scalar(255,0,0)); img(cv::Rect(320,0,320,360)).setTo(cv::Scalar(0,180,255)); cv::Rect r(220,80,200,200); cv::rectangle(img,r,cv::Scalar(255,255,255),2); std::cout << img.cols << "x" << img.rows << " type=" << img.type() << "\n"; cv::imwrite("output_fundamentos.png",img); }
